plugins/source/vercel/README.md

```diff
@@ -0,0 +1,7 @@
+# Vercel Plugin
```

Comment:
We'll need to add `Vercel` to https://github.com/cloudquery/cloudquery/blob/51c68f8e934192a27ac6250a66df62e2b06d9797/.github/styles/Vocab/Base/accept.txt

Reply:
Yeah, it will end with merge conflicts and pain so I'm waiting until the review is over